File: documentation/user/en/query/ordering/examples/constant/entity-primary-key-exact.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeStartsWith("code", "lenovo")
				),
				orderBy(
					entityPrimaryKeyExact(104732, 104718, 105929)
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/constant/entity-primary-key-exact.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeStartsWith("code", "lenovo")
        	),
        	OrderBy(
        		EntityPrimaryKeyExact(104732, 104718, 105929)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);